### PR TITLE
Updating intro-windows-in-k8s for 1.23

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

This change should have went into dev-1.23. 
Sorry about that!

/sig windows
/area docs
/assign @jsturtevant @perithompson @sftim @immuzz

### CODE REVIEW

**Comment by jsturtevant** [comment]
/lgtm 

There is another PR with more changes to pause image here https://github.com/kubernetes/website/pull/30327

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 1]
```diff
@@ -612,12 +612,12 @@ network port spaces). Kubernetes uses pause containers to allow for worker conta
 crashing or restarting without losing any of the networking configuration.
 
 Kubernetes maintains a multi-architecture image that includes support for Windows.
-For Kubernetes v1.22 the recommended pause image is `k8s.gcr.io/pause:3.5`.
+For Kubernetes v1.23 the recommended pause image is `k8s.gcr.io/pause:3.6`.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:615]:
```suggestion
For Kubernetes v{{< skew currentVersion >}} the recommended pause image is `k8s.gcr.io/pause:3.6`.

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 2]
```diff
@@ -661,14 +661,15 @@ On Windows nodes, strict compatibility rules apply where the host OS version mus
 match the container base image OS version. Only Windows containers with a container
 operating system of Windows Server 2019 are fully supported.
 
-For Kubernetes v1.22, operating system compatibility for Windows nodes (and Pods)
+For Kubernetes v1.23, operating system compatibility for Windows nodes (and Pods)
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:664]:
```suggestion
For Kubernetes v{{< skew currentVersion >}}, operating system compatibility for Windows nodes (and Pods)
```
(if this goes stale, it's the Kubernetes project's responsibility to fix that).

**Review by sftim — COMMENTED** [review]
How I'd do it: calculate the version.

**Comment by marosset** [comment]
> How I'd do it: calculate the version.

@sftim - Should I try and do something similar for the pause image version?

File: content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md [hunk 3]
```diff
@@ -612,12 +612,12 @@ network port spaces). Kubernetes uses pause containers to allow for worker conta
 crashing or restarting without losing any of the networking configuration.
 
 Kubernetes maintains a multi-architecture image that includes support for Windows.
-For Kubernetes v1.22 the recommended pause image is `k8s.gcr.io/pause:3.5`.
+For Kubernetes v{{< skew currentVersion >}} the recommended pause image is `k8s.gcr.io/pause:3.6`.
```
Comment (sftim) [inline content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md:615]: We don't have data / shortcodes for the pause image version. This LGTM.

**Comment by marosset** [comment]
/assign @divya-mohan0209 @tengqm 
For review/approval

**Comment by reylejano** [comment]
Preview of updated page:
https://deploy-preview-30947--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/windows/intro-windows-in-kubernetes/

Author of PR is SIG Windows chair. SIG Windows technical lead gave an lgtm in [comment](https://github.com/kubernetes/website/pull/30947#issuecomment-996227034)

/lgtm
/approve

**Pull request merged**